macros/contexts/contextUnits.pl

```diff
@@ -377,6 +382,43 @@ =head2 Working with numbers with units
 quantity is the absolute value of the original quantity.
 
 
+=head2 Differentiation of numbers with units
+
+In order to be able to differentiate a formula that returns a number
+with units, the MathObjects library needs to know the units of the
+variable you are differentiating by.  For example, if you have
+
+    $s = Compute("(3t^2 - 2x) m");
```

Comment:
I think you want `$s = Compute("(3t^2 - 2t) m");`

Reply:
Change made.  Thanks!